tests/unit/test_conf.py

```diff
@@ -225,3 +226,22 @@ def test_max_key_fee_from_args(self):
         args = parser.parse_args(['--max-key-fee', '1.0', 'BTC'])
         c = Config.create_from_arguments(args)
         self.assertEqual(c.max_key_fee, {'amount': 1.0, 'currency': 'BTC'})
+
+    def test_one_of_string(self):
+        parser = argparse.ArgumentParser()
+        Config.contribute_to_argparse(parser)
+
+        with self.assertRaises(ValueError):
+            no_vaid_values = OneOfString([], "no valid values", None)
```

Comment:
these can just be like `self.assertRaises(ValueError, OneOfString, [], "no valid values", None)`

Reply:
no, don't use context-less `self.assertRaises`, using `with self.assertRaises` context manager is the modern/better.
without context manager you don't get any type checking since you're passing arguments which go to `OneOfString` to `assertRaises` instead.